composer.json

```diff
@@ -43,7 +43,7 @@
         "sensio/framework-extra-bundle": "^3.0 || ^4.0 || ^5.0",
         "sllh/php-cs-fixer-styleci-bridge": "^2.0",
         "sonata-project/exporter": "^1.3",
-        "symfony/phpunit-bridge": "^3.3"
+        "symfony/phpunit-bridge": "^3.3.12"
```

Comment:
We will need this change on all bundles to use SymfonyTestListener?

Reply:
Yes, otherwise you get a signature mismatch issue. Not a big deal since this is a dev dependency.